Gemfile

```diff
@@ -96,6 +96,7 @@ end
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem "capybara"
+  gem "capybara_accessible_selectors", git: "https://github.com/citizensadvice/capybara_accessible_selectors", branch: "main"
```

Comment:
Could you provide the context on why we need to source the gem from github please?

Reply:
Because it's maintained by CA, not yet a registered gem. https://github.com/citizensadvice/capybara_accessible_selectors

Reply:
This is likely the only comment holding me back from giving a 👍

Reply:
🤔 I see.